pkgs/data/fonts/martian-mono/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "martian-mono";
+  version = "0.9.1";
+
+  src = fetchzip {
```

Comment:
Perhaps use `fetchFromGitHub`?

Reply:
Hi, thank a lot for your review, and sorry for the delay!
I'm using `fetchzip` because it's fetching the binary font release. Certainly building the font from source could be better, but I'm not familiar with building fonts from glyphs.